src/styles/index.scss

```diff
@@ -1,3 +1,253 @@
+:root {
+  --color: #00acdc;
+}
+
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+}
+
 body {
   margin: 0;
+  display: grid;
+  grid-template: 60px 1fr/ 1fr;
+}
+
+.header {
+  grid-row: 60px;
```

Comment:
The value '60px' for 'grid-row' is not valid. 'grid-row' should specify the start and end lines of the grid item, like '1 / 2'.